packages/autorest.go/src/generator/fake/servers.ts

```diff
@@ -466,6 +466,10 @@ function createPathParamsRegex(method: Method): string {
   // each path param will replaced with a regex capture.
   // note that some path params are optional.
   let urlPath = method.httpPath;
+  // escape any characters in the path that could be interpreted as regex tokens
+  // per the RFC, these are the pchars that also double as regex tokens
```

Comment:
**p**chars?

Reply:
I was looking at [RFC3986](https://www.rfc-editor.org/rfc/rfc3986), I thought it meant "path character".

Reply:
👍 thanks, that's the grammar I was missing. I'd probably put the RFC number here for anyone who wants to go down that rabbit hole

Reply:
Good call, comment updated.